client/src/components/common/AppBar/index.js

```diff
@@ -16,7 +16,9 @@ const AppBar = ({ auth, logoutHandler, UserAvatar, loading, userName }) => {
     <div className="app-bar">
       <MuiAppBar color="transparent">
         <Toolbar className="app-bar__toolbar">
-          <img src={logo} alt="Logo" className="logo" />
+          <a href="https://gazaskygeeks.com/">
```

Comment:
@ha77an Hey! It might be helpful to use `target='_blank'` here so that it will open in a new tab (a good idea when the link is to a different website).

Reply:
yub, I agree with Tom, Adding blank target would be better to direct user to company website without moving from our application. 
```suggestion 
          <a  target="_blank" href="https://gazaskygeeks.com/">
```